defining_policies_and_profiles/master.adoc

```diff
@@ -1,5 +1,5 @@
 = Defining Policies and Profiles
-CloudForms Documentation Team <{product-title}-docs@redhat.com>
+{product-title} Documentation Team <{product-title}-docs@{product-title}.com>
```

Comment:
The team name is right, but that email is wrong. @Fryguy, do we have an entity like {author-email} that we can use? I don't think we can abstract individual segments like that.